pycbc/workflow/core.py

```diff
@@ -1517,8 +1518,7 @@ def to_segment_xml(self, override_file_if_exists=False):
         # create XML doc and add process table
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
-        process = ligolw_utils.process.register_to_xmldoc(outdoc, sys.argv[0],
-                                                          {})
+        process = ligolw_process.register_to_xmldoc(outdoc, sys.argv[0], {})
```

Comment:
I've completely made a new virtualenv and installed everything from scratch, same problem still. When importing glue.ligolw.utils it does not load the submodule process. Looking at the code this is as expected, isn't it?
[`glue/ligolw/utils/__init__.py`](https://github.com/ligo-cbc/pycbc-glue/blob/master/glue/ligolw/utils/__init__.py#L32-L59)